butterknife-compiler/src/main/java/butterknife/compiler/ButterKnifeProcessor.java

```diff
@@ -1273,7 +1326,7 @@ private static boolean isTypeEqual(TypeMirror typeMirror, String otherType) {
         return null;
       }
       typeElement = (TypeElement) ((DeclaredType) type).asElement();
-      if (parents.contains(typeElement)) {
+      if (parents.contains(typeElement) || hasViewBinder(typeElement)) {
```

Comment:
This comment is about all the changes above. Here, the idea is that we want to take into account the situation where a sub class is being processed and not its super class (because we are in an incremental build or because the super class is in a different module).

Even though the super class is not passed to the annotation processor, we can know if butterknife has generated some code for it by testing the presence of the generated class in the classpath. If such a `__Binder` exists for the super class, we create a `StubBindingSet` for it: we do re-scan its annotations here, only because we want to know which constructor of the super class to invoke later when we generate the `__Binder` of the sub class.

Of course, I am open to amend the way of doing things if there is a better solution.